### ISSUE TITLE
Hostname vs IP Address

### ISSUE THREAD

**Issue by jamie-2bit** [troygNZ/homebridge-solax#36]
I'm using the local LAN IP of the inverter in the address field (same as your example code) but it seems to want a URL hostname? 

[1/6/2022, 2:06:49 PM] [Solax Inverter] That did not go well. Error: TypeError: Only absolute URLs are supported
[1/6/2022, 2:06:49 PM] [Solax Inverter] Failed to read from Solax. Error: TypeError: Only absolute URLs are supported
[1/6/2022, 2:06:54 PM] [Solax Inverter] That did not go well. Error: TypeError: Only absolute URLs are supported
[1/6/2022, 2:06:54 PM] [Solax Inverter] Failed to read from Solax. Error: TypeError: Only absolute URLs are supported

Example Code:

"platforms": [
      {
        "platform" : "SolaxHomebridgePlugin",
        "name" : "Solax Inverter",
        **"address": "http://192.168.1.40",**
        "pollingFrequencySeconds": 60,
        "movingAverageHistoryLength": 10,
        "exportAlertThresholds": [-5000, 0, 1000, 1500],
        "showStrings": true,
        "hasBattery": false,
        "valueStrategy": "SimpleMovingAverage"
      }
    ]

**Comment by troygNZ** [comment]
I have seen this warning within the Config UI after updating to a newer version of Homebrdige UI, though never  stopped it from running.
Is this stopping the plugin from working?

**Comment by troygNZ** [comment]
I've updated the config schema, and removed the hostname format requirement. This "hostname" format previously allowed for IP addresses, and traditional hostnames, though the latest verion of Homebridge UI seems to have locked this down.

Interestingly the docs still imply it'll allow for IP addresses with the hostname formatter.
https://developers.homebridge.io/#/config-schema#format

TLDR, it'll now allow you to enter the IP address without it complaining. Released in v 1.3.4
